src/ejabberd_cowboy.erl

```diff
@@ -148,7 +148,7 @@ do_start_cowboy(Ref, Opts) ->
     SSLOpts = gen_mod:get_opt(ssl, Opts, undefined),
     NumAcceptors = gen_mod:get_opt(num_acceptors, Opts, 100),
     TransportOpts0 = gen_mod:get_opt(transport_options, Opts, #{}),
-    TransportOpts = TransportOpts0#{num_acceptros => NumAcceptors},
+    TransportOpts = TransportOpts0#{num_acceptors => NumAcceptors},
```

Comment:
Great catch! Not related to the whole PR, but it fixes #2770, we should resolve this issue after merging.

Reply:
I put it in a different commit to separate it from the rest.